lib/pg_ha_migrations/unsafe_statements.rb

```diff
@@ -80,6 +80,16 @@ def unsafe_add_index(table, column_names, options = {})
       raise PgHaMigrations::InvalidMigrationError, "ActiveRecord drops the :opclass option when supplying a string containing an expression or list of columns; instead either supply an array of columns or include the opclass in the string for each column"
     end
 
+    validated_table = PgHaMigrations::Table.from_table_name(table)
```

Comment:
Subtle, but maybe relevant behavior change here. This will raise a `PgHaMigrations::UndefinedTableError` instead of bubbling up an error from postgres when the table doesn't exist. I _think_ that's okay...

Oh and I think I missed a test for this. Doing that now

Reply:
That seems fine.